src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -74,8 +74,8 @@
                                         x:Uid="Globals_DefaultTerminal"
                                         x:Load="false">
                     <ComboBox x:Name="DefaultTerminal"
-                              ItemsSource="{x:Bind State.Settings.DefaultTerminals}"
-                              SelectedItem="{x:Bind State.Settings.CurrentDefaultTerminal, Mode=TwoWay}"
+                              ItemsSource="{x:Bind AppSettings.DefaultTerminals}"
+                              SelectedItem="{x:Bind AppSettings.CurrentDefaultTerminal, Mode=TwoWay}"

```

Comment:
This one is weird. We need `CascadiaSettings` for defterm. But `Globals` already has a reference to `CascadiaSettings`. For now, I've been passing it in separately to the navigation state and storing a reference to it on the page. Long term, we should be using `CascadiaSettingsViewModel` instead, I think. But idk if this abstraction is right. Thoughts?

Reply:
meh, this is fine, I don't think it's much worse than it was before